### ISSUE TITLE
Unlock database Window frozen on macOS 11.2

### ISSUE THREAD

**Issue by crazyelectron-io** [keepassxreboot/keepassxc#6068]
## Overview
When using the touchID option for unlock, after one or two unlocks, the Unlock Database screen keeps focus (cannot change that) and does not respond to any click or input. The OK button does show green.
Quiting KeepassXC and restarting it solves it temporary.

## Steps to Reproduce
1. Start KeepassXC
2. Unlock DB by providing password
3. Use Global AutoType to fill in a userid/password
3. Lock the DB
4. Use Global AutoType and unlock the DB with TouchID (after clicking OK or pressing ENTER)
5. Select the entry
6. The Unlock KeePassXC Database windows remains in focus and does not respond to any input

Operating System: macOS Big Sur 11.2 (on MacBook Pro M1)

**Comment by droidmonkey** [comment]
To be clear, this only happens when the unlock database pop-up window is used to unlock the database? It does not happen when using the main keepassxc window to unlock?

**Comment by crazyelectron-io** [comment]
Indeed, when unlocking from the main window everything works fine.

**Comment by superlevure** [comment]
Hi, 
I'm having the same issue on a Intel based Mac

**Comment by superlevure** [comment]
Do we have a lead on this bug ? I'm experiencing it at least twice a day, it's really frustrating 

If you need more information I can provide a video of the bug, I'd be happy to give it a shot myself but I have no experience in C++, maybe with some help to point me in the right direction ? 

Thanks for the great tool !

**Comment by tkipkurgat** [comment]
I have both M1 and intel macbook and the issue occurs only on M1. 
I am using keepassXC with brave extension,

**Comment by keaaw** [comment]
I'm a relatively new KeePassXC user on M1 , and after about 2-3 weeks of heavy use with a ton of touch ID unlocks, all of a sudden today touch ID unlock started hanging.  I do not have the firefox extension, but I do have the chrome extension, but I've switched to using firefox.    And I notice that it is not all of my vaults, I have 2 and one of them started to experience the touch ID unlock hang today but the other one still unlocks fine with touch ID.

Update: I disabled browser integration in settings for the vault that was hanging and touch ID unlock started working again.   I enabled browser integration but unchecked "Firefox" and it still works.  I then checked "Firefox" back on and it stayed working.   So browser integration enabled seems to be the likely symptom.
